GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Controller/Garage/BackgroundGaragePageController.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using UnityEngine;
+
+// Controller for background garage page.
+public class BackgroundGaragePageController : MonoBehaviour
+{
+    public GameObject backGroundImages;
+
+    private void OnEnable()
+    {
+        RefreshPages();
+    }
+    
+
+    public void OnItemBlueGrassBackgroundClicked()
+    {
+        SwitchToTargetBackground(BackgroundList.BlueGrassBackground);
+    }
+
+    public void OnItemMushroomBackGroundClicked()
+    {
+        SwitchToTargetBackground(BackgroundList.MushroomBackground);
+    }
+
+
+    public void SwitchToTargetBackground(BackgroundList.Background targetBackground)
+    {
+        GameDataController.GetGameData().ChangeBackground(targetBackground);
+        RefreshPages();
+    }
+
+    public void RefreshPages()
+    {
+        CheckCarOwnership();
+        CheckUsingStatus();
+    }
+
+    // Check if player owns the car.
+    private void CheckCarOwnership()
```

Comment:
Should be renamed to CheckBackgroundOwnership and comment adjusted accordingly?

Reply:
yes you are right, I forgot to update it after I copied from another file.